miniapp/USERGUIDE.md

```diff
@@ -216,11 +216,14 @@ class MiniAppActivity : Activity(), CoroutineScope {
 
 `MiniAppDisplay.navigateBackward` and `MiniAppDisplay.navigateForward` facilitates the navigation inside a mini app if the history stack is available in it. A common usage pattern could be to link it up to the Android Back Key navigation.
 
+**Note:** Clearing up the mini app display is essential. `MiniAppDisplay.destroyView` is required to be called when exit miniapp.
```

Comment:
Maybe we should also update the code sample above to include this? For example:
```
override fun onDestroy() {
    super.onDestroy()
    miniAppDisplay.onDestroy()
}
```

Reply:
I don't know which example is good here. The `onDestroy()` example above is correct for separated miniapp activity case but for the fragment, the example could be something like this 🤔

```
// hostapp example
fun onMiniappRemove() {
   miniAppDisplay.destroyView() // miniapp sdk call
   closeScreen()
}
```

Reply:
I think, the existing line is fine becuase including code example may arise confusion in other clients who have different architecture. Though, we could write `miniAppDisplay.destroyView()` instead of `miniAppDisplay.destroyView`.

Reply:
Yeah I agree on just only description line here.